pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[tool.poetry]
+name = "Heksher"
+version = "0.1.0"
+description = "Heksker"
+authors = ["Biocatch LTD"]
+
+[tool.poetry.dependencies]
+python = "~3.8"
+
+fastapi = "^0.62.0"
+httpx = "^0.16.0"
+orjson = "^3.4.0"
+pydantic = "^1.7.0"
+databases = {version="^0.4.1", extras=['postgresql']}
+uvicorn = "^0.13.2"
+SQLAlchemy = "^1.3.22"
+envolved = "^0.2.0"
+
+alembic = {version="^1.4.3", optional=true}
+psycopg2 = {version="^2.8.6", optional=true}
+
+[tool.poetry.dev-dependencies]
+pytest-asyncio = "*"
+pytest-cov = "^2.10"
+flake8 = "^3.8.4"
+pytype = {version="^2020.12.16", markers = "platform_system != 'Windows'"}
```

Comment:
wow you want to use pytype and somehow it doesn't run on Windows. What is so platform dependent :|